src/WidgetRegistry.ts

```diff
@@ -78,7 +86,7 @@ export class WidgetRegistry extends Evented implements WidgetRegistry {
 			item.then((widgetCtor) => {
 				this.registry.set(widgetLabel, widgetCtor);
 				this.emit({
-					type: `loaded:${widgetLabel}`
+					type: getEventType(widgetLabel)
```

Comment:
The functionality here hasn't changed as part of these changes. 

The type `loaded:${widgetLabel}` is actually a "class" of an event in this instance. If it just emitted `widget:loaded` every single widget would need to keep a registry of widgets that it is waiting on being loaded and use this to determine if the target is one of them before invalidating! 

The registration for the loaded events is actually done in the `RegistryHandler` which once emitted invalidates the widget it's associated to and removes the handle for the listener.

I did notice though I'd missed a location that was still using `loaded:${widgetLabel}` for both `symbol` and `string` labels so have fixed that and added tests.